va/validation-authority.go

```diff
@@ -117,9 +117,14 @@ func (va ValidationAuthorityImpl) validateSimpleHTTP(identifier core.AcmeIdentif
 		// connection immediately.
 		DisableKeepAlives: true,
 	}
+	logRedirect := func(req *http.Request, via []*http.Request) error {
+		va.log.Notice(fmt.Sprintf("validateSimpleHTTP [%s] redirect from %q to %q", identifier, via[len(via)-1].URL.String(), req.URL.String()))
```

Comment:
This isn't getting logged at `Audit`, so that's OK.
That said, I don't think this needs to go at `Notice`, IMO it should log at `Info`.